src/GameController/Enums.cs

```diff
@@ -12,9 +12,9 @@
 using Foundation;
 
 namespace GameController {
-	[Deprecated (PlatformName.MacOSX, 10, 15, message: "GCMicroGamepadSnapshot has been deprecated, use [GCController controllerWithMicroGamepad] instead")]
-	[Deprecated (PlatformName.iOS, 13, 0, message: "GCMicroGamepadSnapshot has been deprecated, use [GCController controllerWithMicroGamepad] instead")]
-	[Deprecated (PlatformName.TvOS, 13, 0, message: "GCMicroGamepadSnapshot has been deprecated, use [GCController controllerWithMicroGamepad] instead")]
+	[Deprecated (PlatformName.MacOSX, 10, 15, message: "Use [GCController ControllerWithMicroGamepad] instead")]
+	[Deprecated (PlatformName.iOS, 13, 0, message: "Use [GCController ControllerWithMicroGamepad] instead")]
+	[Deprecated (PlatformName.TvOS, 13, 0, message: "Use [GCController ControllerWithMicroGamepad] instead")]
```

Comment:
That `[...]` is the ObjC syntax - please use the managed name, like
`Use 'GCController.GetMicroGamePadController()' instead.`

note the `.` at the end :)

Reply:
Oh thank you